src/redux/reducers/notifications.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
is this file used/needed?

Reply:
No, we don't need it. I added this during my first attempt before I decided to use React-Toastify. I will remove this.